modin/pandas/series.py

```diff
@@ -42,8 +36,69 @@
 from . import _update_engine
 
 
-@_inherit_docstrings(pandas.Series, excluded=[pandas.Series.__init__])
+@_inherit_docstrings(
+    pandas.Series, excluded=[pandas.Series.__init__], apilink="pandas.Series"
+)
 class Series(BasePandasDataset):
+    """
+    Modin distributed representation of `pandas.Series`.
+
+    Internally passed data is divided into partitions in order to parallelize
+    computations and utilize the user's hardware as much as possible.
+
+    Inherit common for DataFrames and Series functionality from the
+    `BasePandasDataset` class.
+
+    Parameters
+    ----------
+    data : modin.pandas.Series, array-like, Iterable, dict, or scalar value, optional
+        Contains data stored in Series. If data is a dict, argument order is
+        maintained.
+    index : array-like or Index (1d), optional
+        Values must be hashable and have the same length as `data`.
+    dtype : str, np.dtype, or pandas.ExtensionDtype, optional
+        Data type for the output Series. If not specified, this will be
+        inferred from `data`.
+    name : str, optional
+        The name to give to the Series.
+    copy : bool, default: False
+        Copy input data.
+    fastpath : bool, default: False
+        `pandas` internal parameter.
+    query_compiler : query_compiler
+        A query compiler object to create the Series from.
+
+    Attributes
```

Comment:
I wonder if we really need this part of the docstring... in my opinion we don't need it

Reply:
Ok, section was removed.